src/k8s-extension/HISTORY.rst

```diff
@@ -18,7 +18,7 @@ Release History
 1.1.0
 ++++++++++++++++++
 * Migrate Extensions api-version to 2022-03-01
-* microsoft.azureml.kubernetes: Remove inference private review warning message
+* microsoft.azureml.kubernetes: Remove inference private review warning message, disable service bus by default
```

Comment:
This release entry should not be updated since it is for an older version of `k8s-extension`. Please update with the latest version in the `Release History`

Reply:
Updated.